plugins/hue/fetch_url_v2.py

```diff
@@ -0,0 +1,59 @@
+    # hier die abgefangenen fehlermeldungen in den connections, die auf das fehleritem gemapped werden
+    self._connErrors = ['Host is down', 'timed out', '[Errno 113] No route to host']
+
+
+    def _fetch_url_v2(self, url, auth=None, username=None, password=None, timeout=2, method='GET', headers={}, body=None, errorItem=None):
```

Comment:
This seems to be almost the same as in tools.py implemented except the digest handling. Do you think it would be possible to move this code into the common part?